content/en/blog/_posts/2022-XX-XX-iptables-chains.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
+title: "Kubernetes’s IPTables Chains Are Not API"
+date: 2022-XX-XX
+slug: iptables-chains
+---
+
+### or, “Sorry, You Are Not Invited to SIG Network’s `MASQUERADE` Party”
```

Comment:
A subtitle is OK (we might have a front matter field for this, not sure) but actually I wouldn't use a heading for this. I don't think it helps accessibility.